packages/protocol-kit/src/adapters/ethers/EthersAdapter.ts

```diff
@@ -38,6 +38,21 @@ export interface EthersAdapterConfig {
   signerOrProvider: Signer | Provider
 }
 
+/**
+ * Check if the signerOrProvider is compatible with `Signer`
+ * @param signerOrProvider - Signer or provider
+ * @returns true if the parameter is compatible with `Signer`
+ */
+function isSignerCompatible(signerOrProvider: Signer | Provider): signerOrProvider is Signer {
```

Comment:
Could you move this function to `/src/adapters/ethers/utils/index.ts`?

Reply:
Done